examples/huggingface_glue_imdb_app.yaml

```diff
@@ -13,7 +13,7 @@ setup: |
   git checkout v4.21.0
   pip3 install .
   cd examples/pytorch/text-classification
-  pip3 install -r requirements.txt
+  pip3 install -r requirements.txt --extra-index-url https://download.pytorch.org/whl/cu116
```

Comment:
Nit: add a comment saying SkyPilot's default images on AWS/GCP have CUDA 11.6 (Azure 11.5).

(To remind readers why pinning the version is needed)

Reply:
Good point! Added! Thanks!